### PR TITLE
Add a metrics_from_postfix boolean flag for the metrics role

### PR DESCRIPTION
**Pull request by natoscott**
Tackles Red Hat BZ #2051737

### CODE REVIEW

**Comment by richm** [comment]
Looks like there is a problem on el6

File: tests/tests_sanity_postfix.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  roles:
+    - role: linux-system-roles.metrics
+      vars:
+        metrics_from_postfix: yes
+
+  pre_tasks:
+    - meta: end_host
+      when: (ansible_distribution in ['RedHat', 'CentOS'] and
+             ansible_distribution_major_version|int < 7) or
+             ansible_distribution not in ['Fedora', 'RedHat', 'CentOS']
```
Comment (richm) [inline tests/tests_sanity_postfix.yml:15]: probably better to use `os_family` here to catch (eventually) Rocky Linux, Alma, etc.
```
ansible_os_family != 'RedHat'
```

Comment (natoscott) [reply tests/tests_sanity_postfix.yml:15]: Will do, thanks.  This one's probably better as a separate PR as this pattern is used in a bunch of places.

**Comment by richm** [comment]
[citest bad]

**Comment by richm** [comment]
fedora - at least some of the failures seem due to the VM image running out of space - not sure what to do there in the short term except retry
the other fedora failures are a problem with postfix
```
TASK [Check if postfix pmda is registered] *************************************
task path: /tmp/tmp97jv60op/tests/check_postfix.yml:3
Monday 25 April 2022  23:56:53 +0000 (0:00:00.030)       0:01:22.857 ********** 
fatal: [/cache/fedora-35.qcow2]: FAILED! => {
    "changed": true, 
    "cmd": "pmprobe -I pmcd.agent.status | grep -w '\"postfix\"'", 
    "delta": "0:00:00.009054", 
    "end": "2022-04-25 23:56:52.926989", 
    "rc": 1, 
    "start": "2022-04-25 23:56:52.917935"
}

MSG:

non-zero return code
```
not sure why this check is failing

the rhel-8-y failure looks like a network problem - should retest

**Comment by richm** [comment]
I can reproduce the problem locally:
```
tox -e qemu -- --image-name fedora-34 tests/tests_sanity_postfix.yml
....
TASK [Check if postfix pmda is registered] *************************************
task path: /home/rmeggins/linux-system-roles/metrics/tests/check_postfix.yml:3
Monday 25 April 2022  18:34:27 -0600 (0:00:00.032)       0:01:43.212 ********** 
fatal: [/home/rmeggins/.cache/linux-system-roles/fedora-34.qcow2]: FAILED! => {
    "changed": true,
    "cmd": "pmprobe -I pmcd.agent.status | grep -w '\"postfix\"'",
    "delta": "0:00:00.011956",
    "end": "2022-04-26 00:34:28.118268",
    "rc": 1,
    "start": "2022-04-26 00:34:28.106312"
}

MSG:

non-zero return code
TASK [Check if postfix pmda is registered] *************************************
task path: /home/rmeggins/linux-system-roles/metrics/tests/check_postfix.yml:3
Monday 25 April 2022  18:34:27 -0600 (0:00:00.032)       0:01:43.212 ********** 
fatal: [/home/rmeggins/.cache/linux-system-roles/fedora-34.qcow2]: FAILED! => {
    "changed": true,
    "cmd": "pmprobe -I pmcd.agent.status | grep -w '\"postfix\"'",
    "delta": "0:00:00.011956",
    "end": "2022-04-26 00:34:28.118268",
    "rc": 1,
    "start": "2022-04-26 00:34:28.106312"
}

MSG:

non-zero return code
```

**Comment by richm** [comment]
[citest bad]

**Pull request merged**